js/menus/menuTemplate.js

```diff
@@ -156,9 +171,73 @@ function getTemplate(app) {
                 role: 'front'
             }
         ]
+    } else {
+      // Window menu when Windows.
+        template[2].submenu.push(
+            {
+                label: 'Auto Launch on Windows Startup', 
```

Comment:
why Windows here.  Doesn't this feature work on both mac and windows when using AutoLaunch lib?  has this been tested?